phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
+    level: 8
+    paths:
+        - src
```

Comment:
```suggestion
        - src
        - tests
```

Reply:
Besproken met @Marijn-ehvg in de checkout worden de tests excluded: https://github.com/superbrave/checkout-service/blob/main/phpstan.neon#L16